Clew/Controllers/ViewController.swift

```diff
@@ -1725,6 +1727,7 @@ class ViewController: UIViewController, ARSCNViewDelegate, SRCountdownTimerDeleg
     @objc func feedbackButtonPressed() {
         let storyBoard: UIStoryboard = UIStoryboard(name: "SettingsAndHelp", bundle: nil)
         let popoverContent = storyBoard.instantiateViewController(withIdentifier: "Feedback") as! FeedbackViewController
+        popoverContent.preferredContentSize = CGSize(width: UIScreen.main.bounds.size.width, height: UIScreen.main.bounds.size.height)
```

Comment:
I think three times is a good time to simply create a function to display the popover, rather than manually setting the size every time.

Reply:
I might be misunderstanding something though, so lmk of any disagreement

Reply:
hmm true, not a bad idea... This PR was just modifying these existing popovers with a new size change, but simplifying popover creation could be good for a future PR as it is a pretty different potentially breaking change.